utils/RapidproUtils.ts

```diff
@@ -0,0 +1,217 @@
+import { IHttp, IHttpResponse, IRead } from '@rocket.chat/apps-engine/definition/accessors';
+
+export class RapidproUtils {
+    private readonly MESSAGE_CHUNK_SIZE: number = 640;
+
+    private read: IRead;
+    private http: IHttp;
+    private authToken: string;
+    private baseUrl: string;
+    private closeTicket: string;
+
+    public constructor(read: IRead, http: IHttp, baseUrl: string, authToken: string, closeTicket: string) {
+        this.read = read;
+        this.http = http;
+        this.baseUrl = baseUrl;
+        this.authToken = authToken;
+        this.closeTicket = closeTicket;
+    }
+
+    public getAuthorizationHeader() {
```

Comment:
This method actually returns a headers object - not a separated authorization header although only it's present in the object - , so their name should be `getHeaders`.